go.mod

```diff
@@ -2,6 +2,8 @@ module github.com/azure/azure-dev
 
 go 1.20
 
+replace github.com/AzureAD/microsoft-authentication-library-for-go v1.0.0 => github.com/vhvb1989/microsoft-authentication-library-for-go v1.1.0
```

Comment:
this is only until msal releases a new version

Reply:
Yes, I have already ask @chlowell if they can release a new version.

Reply:
`public.WithOpenURL` is available now in MSAL v1.1.0